metadata.yaml

```diff
@@ -1,14 +1,11 @@
-name: discourse-k8s
+name: discourse
```

Comment:
We can't rename the charm, as there are existing deployments using this name, and also there's a machine charm with the same name (however poorly maintained).

Reply:
Noted